### PR TITLE
Add subtract functionality and test

### PR DESCRIPTION
**Pull request by lpereira89**
Closes issue #1

### CODE REVIEW

File: pythoncalculator/__init__.py [hunk 1]
```diff
@@ -1,2 +1,3 @@
 from .add import add 
 
+from .subtract import subtract
```
Comment (jcbpyle) [inline pythoncalculator/__init__.py]: Missing blank new line

File: subtract.py [hunk 2]
```diff
@@ -0,0 +1,2 @@
+def subtract(x, y)
```
Comment (jcbpyle) [inline subtract.py]: Missing colon on line 1 and missing blank new line at end of file

**Review by jcbpyle — CHANGES_REQUESTED** [review]
Hi @lpereira89, please make sure each file committed has a blank new line at the end of the file and that the subtract function is properly defined

File: pythoncalculator/__init__.py (hunk 1 above)
Comment (lpereira89) [reply pythoncalculator/__init__.py]: solved

File: subtract.py (hunk 2 above)
Comment (lpereira89) [reply subtract.py]: solved

**Review by jcbpyle — APPROVED** [review]
Great job! Still doesn't like the end of file for subtract.py but I can deal with that

**Pull request merged**